src/js/main.js

```diff
@@ -227,8 +269,8 @@ Site.Shapes = {
   playAnimation: function() {
     var _this = this;
 
-    // Check is animating has been stopped; we do this here to fail faster
-    if (!_this.animating) {
+    // Check is animating has been stopped or is at limit; we do this here to fail faster
+    if (!_this.animating || _this.atLimit) {
```

Comment:
though maybe we should be checking instead only for the corners and the middle of top, right, bottom, and left. where you actually are not panning, but in the pan zone. cause here youre looking for anywhere at the limit. I would say something more like `y == 0 && x <= Site.window.width * -2` for the top-right corner and then `x <= Site.window.width * -2 && y == Site.window.height / 2` for the middle of the right limit, etc? does that make sense? if would end up being 8 conditions (4 for the corners and 4 for the middle of each side), but maybe theres a way to simplify it...